juniper_official/Solutions/Traffic-Blackhole/npu-resources.rule

```diff
@@ -1,61 +1,68 @@
-                }
-            }
-            field fpc-name {
-                sensor sensor-npu-resources {
-                    path "/components/component/@name";
-                }
-                type string;
-                description "FPC name";
-            }
-            field property-name {
-                sensor sensor-npu-resources {
-                    path "/components/component/properties/property/@name";
-                }
-                type string;
-                description "Resource name";
-            }
-            field property-value {
-                sensor sensor-npu-resources {
-                    path /components/component/properties/property/state/value;
-                }
-                type integer;
-                description "Resource usage";
-            }
-            rule-properties {
-                version 1;
-                contributor juniper;
-                category basic;
-                supported-healthbot-version 1.0.0;
-                supported-devices {
-                    juniper {
-                        operating-system junosEvolved {
-                            products PTX {
-                                platforms PTX10008 {
-                                    releases 22.2R1 {
-                                        release-support min-supported-release;
-                                    }
-                                }
-                            }
-                        }
-                    }
-                }
-            }
-        }
-    }
+	topic linecard.blackhole {
+		rule npu-resource {
+			keys [ fpc-name property-name ];
+			synopsis "PFE Hardware Resource";
+			description "Rule to fetch NPU resource stats";
+			sensor sensor-npu-resources {
+				synopsis "PFE Hardware Resource definition";
+				description "GNMI based telemetry which collects ASIC NPU resource usage";
+				open-config {
+					sensor-name /junos/system/linecard/npu/memory/;
+					frequency 60s;
+				}
+			}
+			field fpc-name {
+				sensor sensor-npu-resources {
+					path "/components/component/@name";
+				}
+				type string;
+				description "FPC name";
+			}
+			field property-name {
+				sensor sensor-npu-resources {
+					path "/components/component/properties/property/@name";
+				}
+				type string;
+				description "Resource name";
+			}
+			field property-value {
+				sensor sensor-npu-resources {
+					path /components/component/properties/property/state/value;
+				}
+				type integer;
+				description "Resource usage";
+			}
+			rule-properties {
+				version 1;
+				contributor juniper;
+				category advanced;
+				supported-healthbot-version 4.2.0;
+				supported-devices {
+					juniper {
+						operating-system junosEvolved {
+							products PTX {
+								platforms PTX10008 {
+									releases 22.2R1 {
+										release-support min-supported-release;
+									}
+								}
+							}
+							products ACX {
+						                platforms All {
```

Comment:
I hope you have qualified the rule on all ACK models. If not mention only tested models.